packages/react-events/src/Press.js

```diff
@@ -982,4 +982,8 @@ const PressResponder: ReactDOMEventResponder = {
   },
 };
 
-export default React.unstable_createEvent(PressResponder);
+export const Press = React.unstable_createEvent(PressResponder);
+
+export function usePress(props: PressProps): void {
+  React.unstable_useEvent(Press, props);
```

Comment:
I think @sebmarkbage and @acdlite were thinking that if you didn't pass `props`, that the default would be:

```jsx
const [isPressed, updatePressed] = useState(false);
React.unstable_useEvent(Press, { onPressChange: updatePressed });
return isPressed;
```

Same thing for `focus` and `hover`.

Reply:
Magic vs less magic. We'd have to have a return object like PressableState built into the responder module.

Reply:
Having `usePress` and other event hooks be conditionally stateful seems confusing and counter to the general rule of avoiding conditional state in hooks. 

It also means that if I wanted both the `isPressed` state while also observing other behavior, then I'd have to redefine the `isPressed` myself. 

Maybe it should _always_ return the state, and users who don't care about it can just ignore the return value?

Reply:
There's also more than one boolean state in most cases.

Press: pressed, longPressed
Focus: focused, focusVisible, focusWithin, focusVisibleWithin

Reply:
Ah yeah, maybe it is `usePressed` then that they were speaking about.